### DISCUSSION TITLE
novel/%EB%A7%88%EB%B2%95%EC%82%AC-%EC%8B%9C%EC%9E%91%ED%95%A9%EB%8B%88%EB%8B%A4/ep00_09/8/1-1%ED%95%99%EA%B8%B0-1%EC%A3%BC%EC%B0%A8-%EA%B8%88%EC%9A%94%EC%9D%BC-1/

### DISCUSSION THREAD

**Discussion by giscus** [melted-mint/fuwari-edited#3] [Comments]
# novel/%EB%A7%88%EB%B2%95%EC%82%AC-%EC%8B%9C%EC%9E%91%ED%95%A9%EB%8B%88%EB%8B%A4/ep00_09/8/1-1%ED%95%99%EA%B8%B0-1%EC%A3%BC%EC%B0%A8-%EA%B8%88%EC%9A%94%EC%9D%BC-1/

  

https://meltedmint.com/novel/%EB%A7%88%EB%B2%95%EC%82%AC-%EC%8B%9C%EC%9E%91%ED%95%A9%EB%8B%88%EB%8B%A4/ep00_09/8/1-1%ED%95%99%EA%B8%B0-1%EC%A3%BC%EC%B0%A8-%EA%B8%88%EC%9A%94%EC%9D%BC-1/

**Comment by applemint35** [comment]
우리 같이 대학원 가지않을래?

**Reply by meltedmint** [reply]
교수 : 자 자 이리로와
